app/controllers/followers_controller.rb

```diff
@@ -0,0 +1,40 @@
+class FollowersController < ApplicationController
+  def index
+    @follows = Follow.all
+    @articles = Article.all if current_user
+    @followes = current_user.followings if current_user
+  end
+
+  def new
+    @follow = Follow.new
+  end
+
+  def create
+    @follow = Follow.new(follow_params)
+
+    if @follow.save
+      redirect_follow
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @follow = Follow.find(params[:id])
+
+    @follow.destroy
+
+    redirect_follow
+  end
+
+  private
+
+  def follow_params
+    follow = params.permit(:follower_id, :followed_user_id)
```

Comment:
what does `permit`?

Reply:
It is used to set the parameter as allowed and limit what attributes should be allowed in the structure and filter anything else